.github/workflows/ci_webui.yaml

```diff
@@ -42,18 +42,22 @@ jobs:
         run: npm test
 
       - name: Lint code
+        if: ${{ !cancelled() }}
         run: npm run lint
 
       - name: Format code
+        if: ${{ !cancelled() }}
```

Comment:
I'll note that this won't always produce the correct message. For example, it failed at "Lint code" step and that step require code transformation. Then the later "Format code" step will produce wrong info.

Ideally, developers should run these checks locally as it provides the best experience (compared to GHA CI). Also, I don't want to encourage developers relying on GHA CI feedback closely. It is quite costly to run a full set of CI checks (linux, macos, windows, etc). That said, I'll still accept this PR if you strongly insist it will help you.